packages/@aws-cdk/aws-eks/lib/instance-data.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * Used internally to bootstrap the worker nodes
+ * This sets the max pods based on the instanceType created
+ * ref: https://amazon-eks.s3-us-west-2.amazonaws.com/cloudformation/2018-08-30/amazon-eks-nodegroup.yamls
+ */
+export const maxPods = Object.freeze(
```

Comment:
I am hoping to get some time to do a full review. This jumped out at me on the scan, I think you are truncating some numbers here. Unless something changed drastically regarding IP limits on a node.

Reply:
You're right, in formatting some things converting from a Terraform variable I had, I completely butchered the max numbers, they originally came from [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#AvailableIpPerENI), should be an easy fix.

It was never down to the pod, always gave room for -1 to -5 pods, just to be safe depending on the node size.